install

```diff
@@ -739,16 +739,23 @@ if [ "$ignore_dependencies" = "f" ]; then
   fi
 
   if [ -x /usr/bin/apt-get ]; then
-    verbose "\n * Reading requirements-${dist}-${code}.apt..."
-    loudCmd "apt-get -q update"
-    package_list_file="${SCRIPT_DIR}/installer/apt/requirements-${dist}-${code}.apt"
+    verbose "\n * Reading packages.ini files..."
 
-    if [ ! -f "$package_list_file" ]; then
-      echo "ERROR: package file does not exist: $package_list_file" >&2
-      exit 1
-    fi
+    packages_files=(
+      "${SCRIPT_DIR}/"
+      "${SCRIPT_DIR}/api"
+      "${SCRIPT_DIR}/python_apps/airtime_analyzer"
+      "${SCRIPT_DIR}/python_apps/pypo"
+    )
 
-    package_list="$(grep -vE '^\s*#' "$package_list_file" | tr '\n' ' ')"
+    set -e
+    package_list=$(
+      "${SCRIPT_DIR}/installer/packages.py" --format=line "${code}" "${packages_files[@]}" ||
```

Comment:
The script is in `${SCRIPT_DIR}/scripts/packages.py`.

Reply:
Woups.